ui/page/seedbackup/verify_seed.go

```diff
@@ -99,7 +100,7 @@ func (pg *VerifySeedPage) OnNavigatedTo() {
 	for _, word := range seedWords {
 		listGroupSeed = append(listGroupSeed, &layout.List{Axis: layout.Horizontal})
 		index := seedPosition(word, allSeeds)
-		shuffledSeed := pg.getMultiSeed(index, dcr.PGPWordList()) // using allSeeds here modifies the slice
+		shuffledSeed := pg.getMultiSeed(index, allSeeds) // using allSeeds here modifies the slice
```

Comment:
As the comment says, this modifies the passed slice so you need a new one every time. This must be:
```suggestion
		shuffledSeed := pg.getMultiSeed(index, pg.wordSeedType.AllSeeds()) // using allSeeds here modifies the slice
```